examples/go.mod

```diff
@@ -6,3 +6,5 @@ require (
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	k8s.io/klog/v2 v2.0.0-20200324194303-db919253a3bc
 )
+
+replace k8s.io/klog/v2 => ./..
```

Comment:
people will copy this and it will be incorrect for them (`./..` won't point to klog)

Reply:
Should i add a trailing comment? (that gets preserved right?)

Reply:
added trailing comment